spec.html

```diff
@@ -2021,7 +2021,7 @@ <h1>BinaryXor ( _x_, _y_ )</h1>
 
         <emu-clause id="sec-bigintbitwiseop" aoid="BigIntBitwiseOp">
           <h1>BigIntBitwiseOp ( _op_, _x_, _y_ )</h1>
-          <p>The abstract operation BigIntBitwiseOp takes arguments _op_ (a sequence of Unicode code points), _x_, and _y_. It performs the following steps when called:</p>
+          <p>The abstract operation BigIntBitwiseOp takes arguments _op_ (a sequence of Unicode code points), _x_ (a BigInt) and _y_ (a BigInt). It performs the following steps when called:</p>
```

Comment:
```suggestion
          <p>The abstract operation BigIntBitwiseOp takes arguments _op_ (a sequence of Unicode code points), _x_ (a BigInt), and _y_ (a BigInt). It performs the following steps when called:</p>
```

Reply:
whoops, thanks

Reply:
Someday the linter will handle this; it's already on my list.